src/source/Ice/IceAgent.c

```diff
@@ -2272,14 +2298,20 @@ STATUS handleStunPacket(PIceAgent pIceAgent, PBYTE pBuffer, UINT32 bufferLen, PS
             break;
 
         case STUN_PACKET_TYPE_BINDING_RESPONSE_SUCCESS:
-
+            checkSum = COMPUTE_CRC32(pBuffer + STUN_PACKET_TRANSACTION_ID_OFFSET, STUN_TRANSACTION_ID_LEN);
             // check if Binding Response is for finding srflx candidate
             if (transactionIdStoreHasId(pIceAgent->pStunBindingRequestTransactionIdStore, pBuffer + STUN_PACKET_TRANSACTION_ID_OFFSET)) {
                 CHK_STATUS(findCandidateWithSocketConnection(pSocketConnection, pIceAgent->localCandidates, &pIceCandidate));
                 CHK_WARN(pIceCandidate != NULL, retStatus,
                          "Local candidate with socket %d not found. Dropping STUN binding success response",
                          pSocketConnection->localSocket);
 
+                // Update round trip time for serial reflexive candidate
+                pIceAgent->rtcIceServerDiagnostics[pIceCandidate->iceServerIndex].totalResponsesReceived++;
+                CHK_STATUS(hashTableGet(pIceAgent->rtcIceServerDiagnostics[pIceCandidate->iceServerIndex].requestTimestampDiagnostics, checkSum, &requestSentTime));
```

Comment:
Here, it will fail if the item doesn't exist. Are you sure you want to let it fail in this case or you might warn by checking if the item exists hashTableContains I believe is the API to check first